.changeset/config.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Are these configs extensible? Would it be possible and useful to add a base config in a shared location each release group can extend?

Reply:
Unfortunately, they're not extensible as far as I can tell.